### PR TITLE
Back-to-back Vote Tweaks

### PR DESCRIPTION
**Pull request by Geevies**
* The gamemode of the last round can no longer be voted for. Voting for secret unaffected.

FEEDBACK THREAD: https://forums.aurorastation.org/topic/14484-back-to-back-vote-tweaks-feedback

### CODE REVIEW

**Comment by Geevies** [comment]
!review

File: config/example/config.txt [hunk 1]
```diff
@@ -213,6 +213,9 @@ TRANSFER_TIMEOUT 72000
 ## Ghosts who died will still be able to vote.
 #NO_DEAD_VOTE
 
+## uncomment to allow gamemodes to be voted for back-to-back
```
Comment (NonQueueingMatt) [inline config/example/config.txt:216]:
```suggestion
## Uncomment to allow gamemodes to be voted for back-to-back
```

**Comment by Geevies** [comment]
Feedback points to "this is not the way *clickclick*"

File: config/example/config.txt [hunk 2]
```diff
@@ -213,6 +213,9 @@ TRANSFER_TIMEOUT 72000
 ## Ghosts who died will still be able to vote.
 #NO_DEAD_VOTE
 
+## Uncomment to allow gamemodes to be voted for back-to-back
+#VOTE_ALLOW_LAST_GAMEMODE
```
Comment (Geevies) [inline config/example/config.txt:217]:
```suggestion
VOTE_ALLOW_LAST_GAMEMODE
```

**Comment by Geevies** [comment]
!review

**Review by JohnWildkins — APPROVED** [review]

**Comment by NonQueueingMatt** [comment]
Stale, and I don't think this is going to be a good change.

**Pull request closed without merging**